apps/andi/lib/andi/input_schemas/datasets.ex

```diff
@@ -135,13 +135,16 @@ defmodule Andi.InputSchemas.Datasets do
   end
 
   def is_unique?(id, data_name, org_name) do
+    org_name = String.downcase(org_name)
+    data_name = String.downcase(data_name)
+
     get_all()
```

Comment:
I'd love to see this done in a query, as it's likely more performant, but it's not a show stopped

Reply:
Additionally, is there a test for this? That might make it slightly less painful to migrate this to a query later.

Reply:
good point, there are some tests so we can do this